.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "monthly"
```

We should add a newline at EOF.
```suggestion
      # Check for updates to GitHub Actions every week
      interval: "monthly"
      
```

---

Added, thanks!